### PR TITLE
[13.0][MIG] stock_no_negative

### PR DESCRIPTION
**Pull request by qdelcourte**

### CODE REVIEW

File: stock_no_negative/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+11.0.1.1.0 (2018-12-13)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Add the ability to allow negative stock for individual stock locations.
```
Comment (ps-tubtim) [inline stock_no_negative/readme/HISTORY.rst:10]: Please edit `HISTORY.rst` up to date.

File: stock_no_negative/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* Alexis de Lattre <alexis.delattre@akretion.com>
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
+  * Jordi Ballester
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+* Tecnativa <https://www.tecnativa.com>
+  * Pedro M. Baeza
```
Comment (ps-tubtim) [inline stock_no_negative/readme/CONTRIBUTORS.rst:6]: Please add you add a contributor.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@qdelcourte Please use pre-commit to fix error.

**Comment by ps-tubtim** [comment]
@qdelcourte Please generate new README.rst after edit .rst file with https://github.com/OCA/maintainer-tools.

**Review by jesus01x — APPROVED** [review]
functional and technical review.

**Comment by kittiu** [comment]
@ps-tubtim i think generate readme is not necessary. Ocabot will do it when merge.

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by alan196 — APPROVED** [review]
Functional and technical review 👍 

You only need to use pre-commit to fix travis

**Comment by alan196** [comment]
@qdelcourte will you continue working with this PR?

**Review by ps-tubtim — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Well, I guess travis still need to be attended...

**Comment by pedrobaeza** [comment]
Superseded by #666

**Pull request closed without merging**